readme.md

```diff
@@ -13,12 +13,103 @@
         <img src="https://img.shields.io/badge/documentation-read-green?style=flat-square"
             alt="documentation"></a>
 </p>
-        
-**UltiCa** (*Ulti*mate *Ca*taclysm) is a community made tileset for Cataclysm: Dark Days Ahead. It is inspired by old school pixel art games and tries to encapsulate a dark tone and atmosphere of utter desperation of the cataclysm.
 
-This repository stores individual sprites for UltiCa in PNG format, and also source files with layering in *PSD* (Photoshop), *XCF* (gimp 2.10) or *KRA* (Krita) formats. PNGs will then be compiled into a tileset using a python script maintained by mlangsdorf (The script is located in the main Cataclysm repo under `tools/gfx_tools/compose.py`).
+Community-made tilesets for [Cataclysm: Dark Days Ahead](https://github.com/CleverRaven/Cataclysm-DDA).
+Tilesets give the game a different aesthetic.
+
+This repository stores individual sprites for UltiCa in PNG format, and also source files with layering in *PSD* (Photoshop), *XCF* (gimp 2.10) or *KRA* (Krita) formats.
+PNGs will then be compiled into a tileset using [a python script](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/tools/gfx_tools/compose.py) maintained by [mlangsdorf](https://github.com/mlange-42).
```

Comment:
```suggestion
PNGs will then be compiled into a tileset using [a python script](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/tools/gfx_tools/compose.py).
```
While mlangsdorf made the script he hasn't worked on the project for years now, we're definitly not going to pin maintenance on him by linking to his github profil directly